.github/workflows/build-gar-reusable.yml

```diff
@@ -0,0 +1,82 @@
+name: Workflow template for Build Prover builtin Setup Keys
+
+on:
+  workflow_call:
+    inputs:
+      image_tag_suffix:
+        description: "Commit sha or git tag for Docker tag"
+        required: true
+        type: string
+      setup_keys_id:
+        description: "Commit sha for downloading keys from bucket dir"
+        required: true
+        type: string
+      push_asia:
+        description: "Push images to Asia GAR"
+        required: false
+        default: false
+        type: boolean
+
+jobs:
+  build-gar-prover:
+    name: Build GAR prover
+    runs-on: [self-hosted, ci-runner]
+    strategy:
+      fail-fast: false
+      matrix:
+        # https://github.com/matter-labs/gitops-kubernetes/blob/main/apps/base/zksync-v2-config/zksync-v2-config.yaml#L385
```

Comment:
should we expose it?

Reply:
Removed, I hope we have this knowledge somewhere.